client/my-sites/site-settings/theme-setup/theme-setup-card.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import ActionPanel from 'my-sites/site-settings/action-panel';
+import ActionPanelTitle from 'my-sites/site-settings/action-panel/title';
+import ActionPanelBody from 'my-sites/site-settings/action-panel/body';
+import ActionPanelFooter from 'my-sites/site-settings/action-panel/footer';
+import Notice from 'components/notice';
+import Button from 'components/button';
+
+const ThemeSetupCard = ( { translate } ) => (
+	<ActionPanel>
+		<ActionPanelBody>
+			<ActionPanelTitle>{ translate( 'Theme Setup' ) }</ActionPanelTitle>
+			<Notice status={ 'is-warning' } showDismiss={ false }>
+				{ translate( 'This action cannot be undone.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **3** times:
`translate( 'This cannot be undone' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/theme-setup-dialog)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).